ui/lib/pki/addon/routes/issuers/issuer/edit.js

```diff
@@ -1,3 +1,33 @@
-import PkiIssuerDetailsRoute from './details';
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { withConfirmLeave } from 'core/decorators/confirm-leave';
+@withConfirmLeave()
```

Comment:
Weird thing with this route, it wasn't triggering the decorator properly. Didn't have full bandwidth to look into why but there's solid test coverage if it flakes out again